README.rst

```diff
@@ -37,12 +35,11 @@ and we will also need 4 or 5. We therefore know that a minimum vertex cover
 will have at least 3 nodes; it turns out that there must be four.
 
 With a Lagrange parameter that is too weak, the solutions are sets which are
-too small. In this code, the default of the Lagrange parameter is 2. 
-Repeated runs, using the default, showed sets which had size 2 or 3.
+too small. In the ``minimum_vertex_cover`` function from within the ``dwave_networkx`` package, the default of the Lagrange parameter is 2. 
+Repeated runs, using the default, produced sets which had size 2 or 3.
 
-Hence, the Lagrange parameter value of 5 was set large enough to balance the 
-rest of the terms in the QUBO, and to cause the minimum vertex cover conditions
-to all be satisfied for this problem.
+Hence, the Lagrange parameter value of 5 was (1) set large enough to balance 
+the energy term and constraints, and (2) to cause the minimum vertex cover conditions to all be satisfied for this problem.
```

Comment:
Ungrammatical: it parses as "value of 5 was (1) set ...(2) to cause" 
I think you wanted "of 5 was set large enough to (1) balance ... (2) cause the minimum vertex"
Anyway (2) would be clearer as (2) satisfy all minimum vertex cover conditions for this problem
But should 2 not be small enough rather than large enough?

Reply:
I changed the structure here, thanks.